app/services/x509/session_store.rb

```diff
@@ -2,44 +2,31 @@
 # in an out-of-band fashion (using only the session UUID) instead of having access
 # to the user_session from Devise/Warden
 # Should only be used outside of a normal browser session (such as the OpenID Connect API)
-# See X509::Cacher for accessing PII inside of a normal browser session
 module X509
   class SessionStore
```

Comment:
This class was originally a direct clone of `Pii::Cacher` but then a lot of logic was factored out into `OutOfBandSessionAccessor`.

So this is a true-up to be a clone of `Pii::Cacher` again (preserves the interface, keeps the tests working)

Alternatives:
- Just make the same `instance_eval`  change as in the `OutOfBandSessionAccessor`
- (I think I would like to do this in a follow-up) just add `#load_pii`, `#load_x509` methods to the OOB accessor, and remove both `Pii::SessionStore` and `X509::SessionStore` classes

Reply:
reverted this and went with minimal diff noise in 42e831278484d4799cd8af6e036b8137791cf40f